mobile-app/app/screens/AppNavigator/screens/Balances/hooks/TokenPrice.test.tsx

```diff
@@ -0,0 +1,263 @@
+import { ReactElement } from 'react'
+import { Provider } from 'react-redux'
+import { configureStore } from '@reduxjs/toolkit'
+import { renderHook } from '@testing-library/react-hooks'
+import { BigNumber } from '@defichain/jellyfish-json'
```

Comment:
```suggestion
import BigNumber from 'bignumber.js'
```
probably doesn't matter, but let's keep it consistent